RELEASE/scripts/autoscend/auto_combat.ash

```diff
@@ -957,7 +957,9 @@ string auto_combatHandler(int round, monster enemy, string text)
 		return useSkill($skill[Apprivoisez La Tortue], false);
 	}
 
-	if(!in_zelda() && canUse($skill[Gulp Latte]) && (get_property("_latteRefillsUsed").to_int() == 0) && !get_property("_latteDrinkUsed").to_boolean())
+	if((!in_zelda() || my_class() == $class[Vampyre]) &&	//paths that do not use MP
```

Comment:
Maybe this line should instead be `pathHidesMP()`
which calls a boolean that checks for the paths of zelda, vampyre, zombie master, and... what other paths hide the MP stat?